.github/vale/styles/proselint/Airlinese.yml

```diff
@@ -0,0 +1,8 @@
+extends: existence
+message: "'%s' is airlinese."
```

Comment:
What does "airlinese" mean? :o

Reply:
I think it means something like "airline language"

The rules that we add from external (like "google" and "proselint") are automatically and should not changed (they update on every `vale sync`), that's why there are some in that we may not need and are only suggestions or even turned off. 

The rules basically come as a "set" :D and we choose which from them we want to use in `vale.ini`.